### DISCUSSION TITLE
lottie 4.4.1 crash  "InitializableError.invalidInput()"

### DISCUSSION THREAD

**Discussion by Mozzat** [airbnb/lottie-ios#2409] [General]
## Which Version of Lottie are you using?

Lottie [4.4.1]

## Actual Behavior

<img width="1399" alt="截屏2024-05-17 21 07 30" src="https://github.com/airbnb/lottie-ios/assets/17920445/d0eb1747-b074-468b-a5bc-91b983dd3f65">

**Comment by calda** [comment]
This doesn't look like a crash. Do you have Swift error breakpoint enabled? These thrown errors trigger those error breakpoints.
